### DISCUSSION TITLE
YNAB importing

### DISCUSSION THREAD

**Discussion by fdfytr** [firefly-iii/firefly-iii#5033] [General]
Trying to import profile from YNAB to Firefly in Docker and sharing feedback

```
Error(s)

    Line #2:
        (2) Could not find a valid source account when searching for ID "0" or name "".
        (2) Could not find a valid source account when searching for ID "0" or name "".
    Line #3:
        (2) Could not find a valid destination account when searching for ID "0" or name "".
        (2) Could not find a valid destination account when searching for ID "0" or name "".
    Line #4:
        (2) Could not find a valid source account when searching for ID "0" or name "".
        (2) Could not find a valid source account when searching for ID "0" or name "".
    Line #5:
        (2) Could not find a valid destination account when searching for ID "0" or name "".
        (2) Could not find a valid destination account when searching for ID "0" or name "".

Message(s)

    Line #0: Parsed 5 YNAB transactions for further processing.
    Line #1: Created transaction #1: (empty description) (GBP XXX) 
```

Only one savings account was imported.  :(

https://docs.firefly-iii.org/other-data-importers/install/docker/
"Use the pre-defined script" section - all links are broken

Minor bugs with design as per picture
<img width="697" alt="Screenshot 2021-08-30 at 11 27 43" src="https://user-images.githubusercontent.com/61352041/131326008-abbd781b-a8cc-46e9-b929-35d22824be6f.png">

**Comment by JC5** [comment]
Thanks for letting me know the links are broken, I will fix them right now. I'm not sure what's broken with the import, do you have any logs to share?

**Reply by fdfytr** [reply]
I managed to export finally. Basically had to restart the tool and create asset accounts **prior** export itself

**Reply by JC5** [reply]
Sorry about that. Creating those asset accounts felt to obvious to be included in the documentation, which I will pick up right now.
